Allow ClientHello to span multiple QUIC packets (#3045)
quicwg/base-drafts: draft-ietf-quic-transport.md

or treat it as a connection error.

The first packet sent by a client always includes a CRYPTO frame that contains
the start of the first cryptographic handshake message.  The first CRYPTO frame
sent always begins at an offset of 0 (see {{handshake}}).

Note that if the server sends a HelloRetryRequest, the client will send another

> Initial packet.  This Initial packet will continue the cryptographic handshake

Reviewer: ```suggestion
series of Initial packets.  These Initial packets will continue the cryptographic handshake
```

> and will contain a CRYPTO frame with an offset matching the size of the CRYPTO

Reviewer: ```suggestion
and will contain CRYPTO frames starting at an offset matching the size of the CRYPTO
```

> frame sent in the first Initial packet.

Reviewer: ```suggestion
frames sent in the first flight of Initial packets.
```

or treat it as a connection error.

The first packet sent by a client always includes a CRYPTO frame that contains

> the start of the first cryptographic handshake message.  The first CRYPTO frame

Reviewer 2: ```suggestion
the start or all of the first cryptographic handshake message.  The first CRYPTO frame
```

Author: Fixed

CRYPTO frame sent always begins at an offset of 0 (see {{handshake}}).

Note that if the server sends a HelloRetryRequest, the client will send another
series of Initial packets.  These Initial packets will continue the
cryptographic handshake and will contain CRYPTO frames starting at an offset

> matching the size of the CRYPTO frames sent in the first flight of Initial

Reviewer 3: "matching the size" is a bit odd. Perhaps "immediately after the last byte of..."
